packages/create-llama/helpers/python.ts

```diff
@@ -244,14 +245,25 @@ export const installPythonTemplate = async ({
 
     const dataSourceType = dataSource?.type;
     if (dataSourceType !== undefined && dataSourceType !== "none") {
-      const loaderPath =
-        dataSourceType === "folder"
-          ? path.join(compPath, "loaders", "python", "file")
-          : path.join(compPath, "loaders", "python", dataSourceType);
-      await copy("**", enginePath, {
-        parents: true,
-        cwd: loaderPath,
-      });
+      let loaderPath;
+      if (dataSourceType === "file" || dataSourceType === "folder") {
+        const loaderDir = path.join(compPath, "loaders", "python", "file");
+        const dataSourceConfig = dataSource?.config as FileSourceConfig;
+        if (dataSourceConfig.useLlamaParse) {
```

Comment:
instead of deciding which single file to copy, can we use a dedicated folder `"llamaparse"` for LlamaParse?

So we would end up using;
`path.join(compPath, "loaders", "python", "llamaparse")`

Reason: We want to stick to our concept that we copy folders in create-llama

Reply:
Yes, let me update. I've considered llamaparse as a data source then no need to update this code but end-up with this because i wanted it's as an option for file/folder data source. 
Let me reimplement it.